packages/drivers/odsp-driver-definitions/src/tokenFetch.ts

```diff
@@ -5,7 +5,7 @@
 
 /**
  * Represents token response
- * @alpha
+ * @beta
  */
 export interface TokenResponse {
```

Comment:
@sonalideshpandemsft i think this should be being re-exported by the odsp client package

Reply:
or potentially even duplicated. if you decided to duplicate, please change this back to alpha tagging

Reply:
any particular reason that this should be re-exported from the odsp-client package? can it get referenced from the odsp-driver-defs package?

Reply:
Reexporting this would be similar to how azure-client reexports `ITokenResponse` - probably a reasonable starting point.

Reply:
https://github.com/microsoft/FluidFramework/pull/18840